packages/backend/src/models/OrganizationModel.ts

```diff
@@ -25,6 +25,9 @@ export class OrganizationModel {
             organizationUuid: data.organization_uuid,
             name: data.organization_name,
             chartColors: data.chart_colors,
+            defaultProjectUuid: data.default_project_uuid
```

Comment:
do we have this ternary because default project uuid can have `null` instead of `undefined`? I'm never sure exactly how `node-pg` handles nulls in postgres

Reply:
yes, the query returns `null` values.